src/data_info/services/rename.clj

```diff
@@ -25,7 +25,7 @@
                     (move-all cm (:sources data) (:destination data) :user username :admin-users (cfg/irods-admins) :update-fn update-fn)))
         end-fn (fn [async-task failed?]
                  (notifications/send-notification
-                   (notifications/move-notification (:username async-task) (:sources (:data async-task)) [(:destination (:data async-task))] failed?)))]
+                   (notifications/move-notification (:username async-task) (:sources (:data async-task)) (mapv (fn [p] (ft/path-join (:destination (:data async-task)) (ft/basename p))) (:sources (:data async-task))) failed?)))]
```

Comment:
This somewhat dense `mapv` is adding the filename of each path to the end of the destination folder, so we can pass a list of the moved-to paths.

Reply:
For clarity, I would refactor the anonymous function into a named function; maybe something like `src-path->dest-path`.

Reply:
That's probably better, yeah. The line is rather dense and that'd help.